HoloLensCommander/HoloLensCommander/Utilities/WebRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Web.Http;
+using Windows.Web.Http.Headers;
+
+namespace HoloLensCommander
+{
+    public class AppCenterWebRequest
+    {
+        public async Task<string> HttpWebRequest()
+        {
+            HttpClient httpClient = new HttpClient();
+
+            var request = new HttpRequestMessage();
+
+            request.Headers.Accept.Add(new HttpMediaTypeWithQualityHeaderValue("application/json"));
+            request.Headers.Append("X-API-Token", "9b8e356d29f80de93d88fac0252e48f8fbe22e96");
+
+            Uri uri = new Uri("https://api.appcenter.ms/v0.1/apps");
```

Comment:
We may want to read this from settings. I expect it will change given that it is a v0.1 endpoint.

Reply:
@davidkline-ms Same as above, I am curious about best practices for saving API endpoints in a way that they can be easily maintained/updated.

Reply:
The way the app currently handles this is to create a static readonly or const variable that stores the default value (ex: https://api.appcenter.ms/v0.1/apps). If the user enters a different value into the settings dialog, the value is written into the cache. on startup, if there is a value in the cache it is used, otherwise use the default

Reply:
Okay, so saving the apis in a config file and loading it on startup could work here. If the user needs to change the endpoints, they could open the config file in the app's cache and make changes. It seems weird to expose request header names and endpoints through the UI.

Reply:
I agree that it is a bit odd to expose that info in the UI. Maybe that should be something we just document in the issues section to update the code / config file when the endpoint changes.